src/main/java/com/google/devtools/build/lib/runtime/TerminalTestResultNotifier.java

```diff
@@ -213,6 +222,20 @@ private void addToErrorList(
   }
 
   private void printStats(TestResultStats stats) {
+    TestSummaryFormat testSummaryFormat = options.getOptions(ExecutionOptions.class).testSummary;
+    if ((testSummaryFormat == DETAILED) || (testSummaryFormat == TESTCASE)) {
```

Comment:
There's a switch statement above that you didn't update - this is actually causing our internal Java compiler to give an error message. What should the test summary be in the TESTCASE case? (I'm in favor of using TERSE output, but I didn't want to change it unilaterally.)

Alternatively, since you're adding it to the DETAILED format as well, maybe drop the TESTCASE enum?